Sources/MapboxCoreNavigation/RoutesCoordinator.swift

```diff
@@ -58,7 +60,8 @@ final class RoutesCoordinator {
         }
         state = .passiveNavigation
         lock.unlock()
-        setRoutes(nil, completion)
+        // TODO: Is it safe to set the leg index to 0 when unsetting a route?
+        setRoutes(nil, 0, completion)
```

Comment:
Is it safe to set the leg index to 0 when unsetting a route? If not, what should the leg index be in this case?